src/test_l1_dsAudio.c

```diff
@@ -5057,131 +4778,121 @@ void test_l1_dsAudio_positive_dsSetAudioLevel(void) {
 
-	// Step 07: Attempt to set audio level after terminating audio ports
-	result = dsSetAudioLevel(handle[0], audio_level);
-	CHECK_FOR_EXTENDED_ERROR_CODE( result, dsERR_NOT_INITIALIZED, dsERR_INVALID_PARAM );
-	
-	// End of the test
-	UT_LOG("\n Out %s\n", __FUNCTION__);
+		// End of the test
+		UT_LOG("\n Out %s\n", __FUNCTION__);
+	}
 }
 
-
 /**
- * @brief Ensure dsGetAudioMaxDB() retrieves the maximum audio dB level correctly during positive scenarios.
+ * @brief Ensure dsSetAudioAtmosOutputMode() sets the Audio ATMOS output mode correctly during positive scenarios.
  * 
  * **Test Group ID:** Basic: 01@n
- * **Test Case ID:** 81@n
+ * **Test Case ID:** 95@n
  * 
- * **Dependencies:** None@n
+ * **Dependencies:** Audio device should be connected and working properly.@n
  * **User Interaction:** None
  * 
  * **Test Procedure:**@n
  * |Variation / Step|Description|Test Data|Expected Result|Notes|
  * |:--:|-----------|----------|----------|--------------|-----|
  * |01|Call dsAudioPortInit() - Initialize audio ports | | dsERR_NONE | Initialization must be successful |
  * |02|Call dsGetAudioPort() - Get the port handle for all supported audio ports on the platform | type ,  index = [ Loop through kPorts ]  | dsERR_NONE | Valid port handle must be returned for all supported audio ports |
- * |03|Call dsGetAudioMaxDB() by looping through the acquired port handles to get the maximum audio dB of each port in array| handle: [ loop through valid handles ] , maxDb: [pointer to hold the maximum audio dB value] | dsERR_NONE | A valid maximum audio dB value(maximum value can be 180 dB) must be returned |
- * |04|Call dsGetAudioMaxDB() by looping through the acquired port handles to get the maximum audio dB of each port in new  array| handle: [ loop through valid handles ] , maxDb: [pointer to hold the maximum audio dB value] | dsERR_NONE | A valid maximum audio dB value(maximum value can be 180 dB) must be returned |
- * |05|Compare the values and make sure they are equal| | dsERR_NONE | The values must be equal |
- * |06|Call dsAudioPortTerm() - Terminate audio ports | | dsERR_NONE |  Termination must be successful |
+ * |03|Call dsSetAudioAtmosOutputMode() - by looping through the acquired port handles to enable Audio ATMOS output mode | handle = [valid handle], enable = [TRUE] | dsERR_NONE | Atmos mode must be enabled for all supported audio ports |
+ * |04|Call dsSetAudioAtmosOutputMode() - by looping through the acquired port handles to disable Audio ATMOS output mode | handle = [valid handle], enable = [FALSE] | dsERR_NONE | Atmos mode must be enabled for all supported audio ports |
+ * |05|Call dsAudioPortTerm() - Terminate audio ports | | dsERR_NONE |  Termination must be successful |
  * 
  */
-void test_l1_dsAudio_positive_dsGetAudioMaxDB(void) {
+void test_l1_dsAudio_positive_dsSetAudioAtmosOutputMode(void) {
 	// Start of the test
-	gTestID = 81;
+	gTestID = 77; // Assuming Test Case ID is 238
```

Comment:
Id in comment is wrong. remove the comment. Check other places also

Reply:
done